scripts/pre_push_hook_test.py

```diff
@@ -493,6 +509,24 @@ def mock_start_python_script(unused_script):
         self.assertTrue(
             'Push aborted due to failing frontend tests.' in self.print_arr)
 
+    def test_does_diff_include_travis_yaml_or_protractor_conf_file_failure(self):
```

Comment:
I'm a bit confused about this test. Could you explain what exactly is being tested here? (Again the issue is that it's not clear what specific "failure" behaviour is actually being tested.)

Reply:
Yeah right. Done